docs/robots/boxer/integration_boxer.mdx

```diff
@@ -1,12 +1,81 @@
 ---
-title: Integration
+title: Boxer Integration
+sidebar_label: Integration
 sidebar_position: 2
+toc_min_heading_level: 2
+toc_max_heading_level: 4
 ---
 
 import ComponentIntroductionBoxer from "/components/introduction_boxer.mdx";
+import ComponentCommonSoftwareIntegration from "/components/common_software_integration.mdx";
+import Support from "/components/support.mdx";
 
 <ComponentIntroductionBoxer />
 
-:::warning todo—under construction
+To attach custom hardware to Boxer, you will have to take care of mechanical mounting, electrical
+supply, and software integration. This guide aims to equip you with respect to these challenges.
 
-:::
+---
+
+### Mechanical Mounting
+
+When determining mechanical mounting, you can use the Standard mounting pattern on the top plate of the Boxer.
+At this time, the "PACS" mounting system is not available for Boxer.
+
+#### Mechanical, Standard
+
+External payloads can be attached to the mounting plate of the Boxer by using the predrilled Ø4.2 mm thru holes, using the Ø8 mm countersunk through holes on the underside, or adding new holes.
```

Comment:
This should be discussed with Adrian, who is the latest designer of plate 024925.
The Ø4.2 thru holes don't seem useful, since they are pressed against the top of the Boxer.


The corner Ø8.2 holes for M8-flat-head screws are useful.

Reply:
Following up with Adrian on Slack.